sequelizeDatabase/sequelFunctions.js

```diff
@@ -1,350 +1,481 @@
+      Password: Password
+    }
+  })
+    .spread((user, createdBoolean) => {
+      return user;
+    })
+    .catch(err => console.error('Error in "insertUser", ', err));
 };
 
 insertFund = (FundName, UserID) => {
-    return models.Funds.create({ //TODO: Make this a findOrCreate to prevent duplicate names.
-        FundName: FundName,
-        UserID: UserID
-    }).then((createdFund) => {
-        return createdFund;
-    }).catch(err => console.error('Error in "insertFund", ', err));
+  return models.Funds.create({
+    //TODO: Make this a findOrCreate to prevent duplicate names.
+    FundName: FundName,
+    UserID: UserID
+  })
+    .then(createdFund => {
+      return createdFund;
+    })
+    .catch(err => console.error('Error in "insertFund", ', err));
 };
 
 insertSharedFunds = (FundID, UserID) => {
-    //NOTE: UserID here is the user the fund is being shared with, not the fund owner.
-    return models.SharedFunds.create({//TODO: May need a "findOrCreate" with both fields in 'where'.
-        FundID: FundID,
-        UserID: UserID
-    }).then((createdSharedFund) => {
-        return createdSharedFund;
-    }).catch(err => console.error('Error in "insertSharedFunds", ', err));
+  //NOTE: UserID here is the user the fund is being shared with, not the fund owner.
+  return models.SharedFunds.create({
+    //TODO: May need a "findOrCreate" with both fields in 'where'.
+    FundID: FundID,
+    UserID: UserID
+  })
+    .then(createdSharedFund => {
+      return createdSharedFund;
+    })
+    .catch(err => console.error('Error in "insertSharedFunds", ', err));
 };
 
 insertPerson = (FundID, Name, LinkedInUrl, Comments) => {
-    return models.Individuals.findOrCreate({where:{LinkedInUrl: LinkedInUrl, FundID: FundID}, 
-        defaults: {
-            Name: Name,
-            Comments: Comments}    
-    }).spread((user,createdBoolean) => {
-        return user;
-    }).catch(err => console.error('Error in insertPerson', err));
+  return models.Individuals.findOrCreate({
+    where: { LinkedInUrl: LinkedInUrl, FundID: FundID },
+    defaults: {
+      Name: Name,
+      Comments: Comments
+    }
+  })
+    .spread((user, createdBoolean) => {
+      return user;
+    })
+    .catch(err => console.error("Error in insertPerson", err));
 };
 
 insertCompany = (CompanyName, FundID) => {
-    return models.Companies.findOrCreate({where: {CompanyName: CompanyName, FundID: FundID}
-    }).spread((company, createdBoolean) => {
-        //console.log('Company Created: ', company);
-        return company;
-    }).catch(err => console.error('Error in insertCompany',err));
+  return models.Companies.findOrCreate({ where: { CompanyName: CompanyName, FundID: FundID } })
+    .spread((company, createdBoolean) => {
+      //console.log('Company Created: ', company);
+      return company;
+    })
+    .catch(err => console.error("Error in insertCompany", err));
 };
 
 insertOriginalFundPosition = (IndividualID, CompanyID, PositionName) => {
```

Comment:
I'd forgotten we had updated this table. We don't need to add any new field to Companies.